src/main/scala/io/viash/config/dependencies/ViashhubRepositoryTrait.scala

```diff
@@ -47,10 +47,14 @@ trait ViashhubRepositoryTrait extends AbstractGitRepository {
   def getCacheIdentifier(): Option[String] =
     Some(s"viashhub-${fullRepo.replace("/", "-")}${tag.map(_.prepended('-')).getOrElse("")}")
 
-  val uri = s"https://packages.viash-hub.com/$fullRepo.git"
-  lazy val uri_ssh = s"git@packages.viash-hub.com:$fullRepo.git"
+  // TODO: replace by using anc
+  val server = sys.env.getOrElse("VIASH_VSH_HOST", "packages.viash-hub.com")
```

Comment:
What about `VIASH_HUB_HOST`? 

Or `VIASH_HUB_GIT_HOST`, since this only pertains to where the git repos are stored.